modules/enabling-cluster-capabilites.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// *post_installation_configuration/cluster-capabilities.adoc
+
+[id="enabling_the_cluster_capabilities_{context}"]
+= Enabling the cluster capabilities
+
+As a cluster administrator, you can enable the capabilities in the cluster by using OpenShift CLI (oc).
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (oc).
+* You have installed the jq package.
```

Comment:
We need to point folks to the  jq package home page otherwise it is confusing.

Reply:
@LalatenduMohanty sure. Please share the jq package home page URL.

Reply:
https://stedolan.github.io/jq/

Reply:
done.